rewrite-java/src/main/java/org/openrewrite/java/style/TabsAndIndentsStyle.java

```diff
@@ -33,6 +33,14 @@ public class TabsAndIndentsStyle implements JavaStyle {
     Integer continuationIndent;
     Boolean indentsRelativeToExpressionStart;
 
+    MethodDeclarationParameters methodDeclarationParameters;
```

Comment:
IntelliJ setting exists in `WrappingAndBracesStyle`, but it doesn't fit well with alignment, so I added the setting in `TabsAndIndentsStyle`